include/deal.II/fe/fe_tools.h

```diff
@@ -841,148 +841,234 @@ namespace FETools
 
 
   /**
-   * Take vectors of finite elements and multiplicities and multiply out
-   * how many degrees of freedom the composed element has per vertex,
-   * line, etc.
-   *
-   * If @p do_tensor_product is true, the number of components
-   * returned in the FiniteElementData object is the sum over the
-   * product of the number of components in each of the finite
-   * elements times the corresponding multiplicity.  Otherwise the
-   * number of components is taken from the first finite element with
-   * non-zero multiplicity, and all other elements with non-zero
-   * multiplicities need to have the same number of vector components.
+   * A namespace that contains functions that help build more
+   * complex finite elements from simpler ("base") elements.
+   *
+   * There are generally two ways in which one can build more complex
+   * elements, and this is reflected by several of the functions in
+   * this namespace having arguments called
+   * <code>do_tensor_product</code>:
+   *
+   * <ol>
+   * <li> Tensor product construction (<code>do_tensor_product=true</code>):
+   * The tensor product construction, in the simplest case, builds a
+   * vector-valued element from scalar elements (see
+   * @ref vector_valued "this documentation module" and
+   * @ref GlossComponent "this glossary entry" for more information).
+   * To give an example, consider creating a vector-valued element with
+   * two vector components, where the first should have linear shape
+   * functions and the second quadratic shape functions. In 1d, the
+   * shape functions (on the reference cell) of the base elements are then
+   * @f{align*}
+   *   Q_1 &= \{ 1-x, x \},
+   *   \\  Q_2 &= \{ 2(\frac 12 - x)(1-x), 2(x - \frac 12)x, 4x(1-x) \},
+   * @f}
+   * where shape functions are ordered in the usual way (first on the
+   * first vertex, then on the second vertex, then in the interior of
+   * the cell). The tensor product construction will create an element with
+   * the following shape functions:
+   * @f{align*}
+   *   Q_1 \times Q_2 &=
```

Comment:
I've taken a look and we seem to consistently use `\times` in this context. I can't seem to find a use of `\otimes` in the context of forming the tensor product space in our documentation. So I would suggest leaving things as is.